tests/blockchain/rbac_client.py

```diff
@@ -127,27 +127,33 @@ def propose_add_role_admins(
         return self._client.get_statuses([signature], wait=10)
 
     def confirm_add_role_admins(self, key, proposal_id, role_id, user_id, reason):
+        data_blob = {
+            'reason': reason
+        }
         batch_list, signature = role_transaction_creation.confirm_add_role_admins(
             txn_key=key,
             batch_key=self._key,
             proposal_id=proposal_id,
             role_id=role_id,
             user_id=user_id,
-            reason=reason,
+            data_blob=data_blob,
```

Comment:
@kthblmfld I agree that this naming lacks consideration. I will try to come up with a better one. The reason for this "data_blob" is that this is the single point of entry to create many different type of transactions. Not all type of transaction creation would take the same parameters. So instead of having a long list of parameters to each transaction creation method, having a dictionary here seems the right way to go. But as you said, it could be named better. I am thinking "params"